webui.py

```diff
@@ -13,6 +13,14 @@
 CMD_FLAGS = '--chat --model-menu'
 
 
+# Allows users to set flags in "OOBABOOGA_FLAGS" environment variable
+if "OOBABOOGA_FLAGS" in os.environ:
+    CMD_FLAGS = os.environ["OOBABOOGA_FLAGS"]
+    print("\33[1;32mFlags have been taken from enivroment Variable 'OOBABOOGA_FLAGS'\33[0m") 
+    print(CMD_FLAGS)
+    print("\33[1;32mTo use flags from webui.py remove 'OOBABOOGA_FLAGS'\33[0m") 
```

Comment:
This can be simplified like this.
```suggestion
CMD_FLAGS = '--chat --model-menu'

# Allows users to set flags in "OOBABOOGA_FLAGS" environment variable
CMD_FLAGS = os.environ.get("OOBABOOGA_FLAGS", CMD_FLAGS)
```
Or combined like this.
```suggestion
CMD_FLAGS = os.environ.get("OOBABOOGA_FLAGS", "--chat --model-menu")
```

Reply:
The console warning that flags are being set from environment variables is essential in case users have OOBABOOGA_FLAGS but are expecting the inline flags to be used by the code.